ansible/roles/alloy/tasks/main.yml

```diff
@@ -0,0 +1,45 @@
+---
+
+- name: Add apt signing key for Grafana repo
+  ansible.builtin.apt_key:
+    url: "{{ alloy_grafana_signing_key }}"
+    state: present
+    id: "{{ alloy_grafana_signing_key_fingerprint }}"
+    keyring: "/etc/apt/trusted.gpg.d/grafana.gpg"
+  tags:
+    - role::alloy
+
+- name: Add Grafana apt repository to apt lists
+  ansible.builtin.apt_repository:
+    repo: "{{ alloy_grafana_repository }}"
```

Comment:
This confuses me slightly, I agree we should add this if it is required but it currently works without this. Is this because I am adding the key to the trusted directory as opposed to a separate keyring?

Or is this just further restricting the keys that are allowed to sign for this repository as a security measure?

Reply:
Gotcha, updated in 61e604d09b663834cd1781492bb4351e57a60aa2.

I have tidied up the old keys from the previously configured directory and updated the sources list accordingly in the role.